scripts/generate-json-schema.ts

```
@@ -0,0 +1,307 @@
+async function runLint(files: string[]) {
+	return Result.try({
+		try: $`bun run lint --fix ${files}`,
+		catch: error => error,
+	});
+}
+
+async function writeFile(path: string, content: string) {
+	return Result.try({
+		try: Bun.write(path, content),
+		catch: error => error,
+	});
```

ESLint

[error] Unsafe return of a value of type error.

(ts/no-unsafe-return)